docs/API/content-type/creating-ctd.md

```diff
@@ -1,19 +1,307 @@
+- **metaDefinition** - contains additional properties of the fields
+
+For simplicity - this example uses a straightforward Content Type, which only has two attributes:
+
+- title
+- postContent
+
+and you can see those listed under the `schemaDefinition` property.
+
+!!! note
+    The `schemaDefinition` part of the JSON payload is based on the bare JSON OpenAPI 3.0 Schema and is fully compatible. 
+    It holds information about the properties of the 
+    <abbr title="Content Type - a model of data that has been defined inside the Content Repository.">CT</abbr> 
+    (in the `properties` key), its types and which properties are required (`required` key). 
+    It always should have `"type": "object"`, as it is an object, and `"additionalProperties": false` 
+    to ensure that API users will not post garbage to the objects of this CTD.
+
+    To ensure that all objects have a proper `id` property 
+    <abbr title="Content Type Definition - a JSON payload that defines the Content Type, it's validation rules, etc.">CTD</abbr> 
+    also should have information about the connection with `AbstractContentTypeSchemaDefinition` using:
+    ```
+    "allOf": [
+        {
+            "$ref": "#/components/schemas/AbstractContentTypeSchemaDefinition"
+        }
+    ],
+    ```
+
+    All property names should have only `a-z`, `A-Z`, `0-9` and `_` characters. `id` and `objectType` property names are restricted.
+
+Finally - the `metaDefinition` attribute is used to store additional properties, which for example, tell us how to 
+render forms for the Content Object; it holds information about the order of the properties (`order` key, which 
+should contain all properties of the object), and about type and validation of the field. It also includes information 
+on relations of the object with other Content Types. 
+
+In this case, Blog Post will have `title` property which will be unique and will render as text input in the CMS panel 
+and `postContent`, which can be duplicated and presented as CKEditor input in the CMS panel. 
+The first input in the form in the CMS panel will be `title` input, and the second will be `postContent`.
+
+!!! Example
+
+    === "CURL"
+    
+        ```
+        curl -X POST "https://api.flotiq.com/api/v1/internal/contenttype" -H 'accept: */*' -H 'X-AUTH-TOKEN: YOUR_API_KEY' -H 'Content-Type: application/json' --data-binary '{"name":"blogposts","label":"Blog Posts","schemaDefinition":{"type":"object","allOf":[{"$ref":"#/components/schemas/AbstractContentTypeSchemaDefinition"},{"type":"object","properties":{"title":{"type":"string"},"postContent":{"type":"string"}}}],"required":["title","postContent"],"additionalProperties":false},"metaDefinition":{"propertiesConfig":{"title":{"label":"Title",inputType":"text","unique":true},"postContent":{"label":"Post content","inputType":"richtext","unique":false}},"order":["title","postContent"]}}'
```

Comment:
When i use this example code i get: `{"data":["Invalid json message received"]}`
There is a typo here
![Hotfix-resolving-reported-issues-by-CiotkaCierpienia-·-Pull-Request-60-·-flotiq-flotiq-docs](https://user-images.githubusercontent.com/645716/116518042-0d049480-a8d0-11eb-92a8-e1b4817f8578.png)